dsa/index.json

```
@@ -146,6 +146,42 @@
             "url": "https://web.archive.org/web/20170801080631/http://cs.brown.edu/~rt/gdhandbook/",
             "type": "book",
             "level": "everyone"
+        },
+        {
+            "title": "Introduction to Algorithms, Second Edition",
+            "url": "https://athena.nitc.ac.in/summerschool/Files/clrs.pdf",
+            "type": "book",
+            "level": "Intermediate"
```

Please use lowercase version of "intermediate".

```suggestion
            "level": "intermediate"
```